docs/source/examplecleanup.rst

```diff
@@ -42,11 +42,13 @@ More on this later!)
 #. Use ``tartufo`` to scan your repository and find any secrets in its history!
 
    Scan your repo!
+   .. note:: The `--fetch/--no-fetch` option for local scans controls whether the local clone is refreshed before scan.
+   This should be set to `no-fetch` for this process.
```

Comment:
```suggestion

   .. note:: The `--fetch/--no-fetch` option for local scans controls whether the local clone is refreshed before scan.
      This should be set to `no-fetch` for this process.
```
Added a blank line before the directive so it will render properly, and added some indentation to stop Sphinx from complaining. 😄 

Sphinx/rST can be super picky. The only way to be really absolutely certain everything renders right is to build the docs, which can be done with either `tox -e docs` or `make -C docs clean html`

Reply:
I wonder if we should also explicitly call out that it's an argument to `scan-local-repo` and not an argument to `tartufo` itself? Just in case people add it in the wrong spot and get confused.